### PR TITLE
More consistent border radii

### PR DESCRIPTION
**Pull request by tilleryd**
Hooking up tabs border radius to global variable

Connecting alert border radius to global border radius

Connecting badge border radius to global border radius

Connecting block text elements border radius to global border radius

Connecting form field border radius to global border radius

Connecting nav element border radius to global border radius

Connecting nabber border radius to global border radius

Connecting pagination button border radius to global border radius

Connecting panel border radius to global border radius

### CODE REVIEW

**Comment by tilleryd** [comment]
FYI @ChrisL1200 @alecholmez

**Comment by tilleryd** [comment]
👍  Looks good.  I just have one comment.  I see a mix of 2 spaces and tabbing for indentation.  I think we should stick to 2 spaces.  That's the standard in other Decipher projects using LESS and SASS.  I believe in Atom there's a user settings configuration where you can set this up.

**Comment by shanberg** [comment]
Yeah, I believe that's because I've ignored the inconsistent formatting between UIKit base and the AlmostFlat theme I've copied some code from.

**Comment by tilleryd** [comment]
Ok, looks good.  Please squash your commits and I'll merge.  You just need to run `reviewPrep` again on this branch and force push.

File: custom/decipher/tab.less [hunk 1]
```diff
@@ -38,4 +40,32 @@
 // Miscellaneous
 // ========================================================================
 
-.hook-tab-misc() {}
\ No newline at end of file
+.hook-tab-misc() {
+
+    /* Modifier: `uk-tab-bottom'
+     ========================================================================== */
```
Comment (tilleryd) [inline custom/decipher/tab.less]: Looks like this is still tabbed.

**Comment by ChrisL1200** [comment]
LGTM

**Pull request merged**